client/src/Component/homepage/homepage.js

```diff
@@ -0,0 +1,116 @@
+import React,{useState,useEffect} from "react";
```

Comment:
You can create a new component `client/src/components/homepage/homepage.js` this path, check `components` spelling.

Reply:
or if you found already component of homepage then delete it :)